src/main/java/AlexLink/HomeWork/String/TextFormatter.java

```diff
@@ -0,0 +1,31 @@
+package AlexLink.HomeWork.String;
+
+import java.util.Arrays;
+
+public class TextFormatter {
+    private static String pattern = "\\s|([.,!?-]\\s)|\\.";
```

Comment:
private static final String PUNCTUATION_MARKS_PATTERN = ....

Reply:
fixed it

Reply:
@qpuhuk You didn't push your changes

Reply:
All done. I'm not finished it at that moment. Thanks a lot